release/release.go

```diff
@@ -75,13 +78,18 @@ func run(name string, args ...string) (string, error) {
 }
 
 func getVersion() string {
-	desc, err := run("git", "describe")
+	desc, err := run("git", "describe", "--dirty")
```

Comment:
differentiate between a build run with a clean git status and one run with uncommitted changes